mex/model/build_release_zip.py

```diff
@@ -0,0 +1,59 @@
+import json
```

Comment:
can we move this to `scripts/build_release_zip.py` so it's not included in the wheel?

Reply:
moved in f68543ef60ba6831ae43a7a371af7a253952e1e2